packages/amplication-prisma-db/prisma/migrations/20221204152837_remove_unused_types/migration.sql

```diff
@@ -0,0 +1,14 @@
+/*
+  Warnings:
+
+  - The values [Flow,ConnectorRestApi,ConnectorRestApiCall,ConnectorSoapApi,ConnectorFile,EntityApi,EntityApiEndpoint,FlowApi,Layout,CanvasPage,EntityPage,Document] on the enum `EnumBlockType` will be removed. If these variants are still used in the database, this will fail.
+
+*/
+-- AlterEnum
+BEGIN;
+CREATE TYPE "EnumBlockType_new" AS ENUM ('ServiceSettings', 'ProjectConfigurationSettings', 'Topic', 'ServiceTopics', 'PluginInstallation', 'PluginOrder');
+ALTER TABLE "Block" ALTER COLUMN "blockType" TYPE "EnumBlockType_new" USING ("blockType"::text::"EnumBlockType_new");
```

Comment:
we need to add another step here to make sure there are not existing records with the old data before changing the type (in case there are, we can delete them because it is probably garbage left from internal tests)